index.bs

```diff
@@ -192,6 +192,7 @@ interface ImageCapture {
     readonly attribute MediaSettingsRange         imageHeight;
     readonly attribute MediaSettingsRange         imageWidth;
     readonly attribute FrozenArray&lt;FillLightMode> fillLightMode;
+    readonly attribute FrozenArray&lt;DOMString>     supportedImageFormats;
```

Comment:
At the moment, `PhotoCapabilities` and `PhotoSettings` have same attribute names, even for arrays. For example `PhotoCapabilities.fillLightMode` represents supported fill light modes (`supportedFillLightModes`).

Question is that, what naming style should be used? If I follow current naming style, then `PhotoCapabilities` should have `FrozenArray<DOMString> imageFormat` attribute and `PhotoSettings` should have `DOMString imageFormat` attribute.

As the attribute names for `PhotoCapabilities` and `PhotoSettings` are the same (currently), implementation-status.md has just one column (`Photo{Capabilities/Settings}`).
If `PhotoCapabilities` and `PhotoSettings` would have different attribute names, it could be confusing and look like both have [`imageFormat` and `supportedImageFormats`](https://github.com/w3c/mediacapture-image/pull/185/commits/daa6ab6cc5f0dfe92936f4e3ee678fd2eb15e691) attributes.

Do you think [implementation-status.md](https://github.com/w3c/mediacapture-image/pull/185/commits/daa6ab6cc5f0dfe92936f4e3ee678fd2eb15e691) looks fine with my modifications?